### PR TITLE
142 update milvus integration

### PR DESCRIPTION
**Pull request by mdswift**
Add requirements.txt file for the Milvus python script
Add details on the 2 Milvus ports and which will be used for the assistant extension

### CODE REVIEW

File: integrations/extensions/starter-kits/language-model-conversational-search/README.md [hunk 1]
```diff
@@ -115,17 +115,17 @@ Follow the steps in the following sections for the custom extensions setup befor
...
+1. Follow the instructions at [Milvus](https://milvus.io) to [install it](https://milvus.io/docs/install_standalone-docker.md) somewhere that your assistant can access it. You will need the server variables (ip-address and REST port) to connect it with your assistant. One option for where to host is an [IBM virtual server](https://www.ibm.com/cloud/virtual-servers). Another is [IBM code engine](https://www.ibm.com/cloud/code-engine). However, it can also be installed on a separate machine as long as it is connected to the internet. Alternatively, Milvus can be run on a different cloud such as AWS. You need to record the IP address of the machine where it's installed Milvus and the port where it is running. Note that there are 2 Milvus ports. One port is for the REST API which is the one used for the assistant extension connection, and it defaults to 9091. The other port is for grpc requests used for indexing the documents, and it defaults to 19530.
```
Comment (jwm4) [inline integrations/extensions/starter-kits/language-model-conversational-search/README.md:118]: grpc should be gRPC

**Review by jwm4 — CHANGES_REQUESTED** [review]
Tiny copy editing change requested.

File: integrations/extensions/starter-kits/language-model-conversational-search/README.md (hunk 1 above)
Comment (mdswift) [reply integrations/extensions/starter-kits/language-model-conversational-search/README.md]: Done!

**Review by jwm4 — APPROVED** [review]
Looks good!

**Pull request merged**